internal/commands/util/utils.go

```diff
@@ -159,14 +158,17 @@ func IsDirOrSymLinkToDir(parentDir string, fileInfo fs.FileInfo) bool {
 			return false
 		}
 
+		if !filepath.IsAbs(realPath) {
```

Comment:
it's not too risky as it's for all files? is it not a possible case in other cases too?
and please leave comment of what it does/ have it in a private method with meaningful name

Reply:
We enter to this line only when it's symlink file